workflow/scripts/flow_generator.py

```diff
@@ -0,0 +1,60 @@
+
+from jinja2 import Environment, FileSystemLoader # templating engine
+import pandas as pd, os
+
+def get_parameters(data, name, crs=None):
+    if name != "bounding_box":
+        return ",".join(list(data[name].values.astype(str)))
+    elif name == "bounding_box" and crs is not None:
+        return get_bounding_box_list(data, crs)
+    
+def generate_bounding_box(row, crs):
+    if crs == "epsg3413":
+        columns = ["left_x", "top_y", "right_x", "lower_y"]
+    elif crs == "wgs84":
+        columns = ["top_left_lat", "top_left_lon", "lower_right_lat", "lower_right_lon"]
+    return "@".join(list(row[columns].values.astype(str)))
+
+def get_bounding_box_list(data, crs="wgs84"):
+    f = lambda x: generate_bounding_box(x, crs) 
+    return ",".join(data.apply(f, axis=1).values)
+
+def generate_cylc_file(csvfile="site_locations.csv", template="flow_template.j2", template_dir="./config", crs="wgs84", minfloearea=350, maxfloearea=90_000):
+    """
+    Generate cylc files from template.
+
```

Comment:
Include the list of column names since those are hard-coded

Reply:
@danielmwatkins But column names would be hardcoded in the csv file, right? I think an assumption needs to be made about the expected column names and perhaps validate the assumption is met before proceeding.

Reply:
The filename must also be `site_locations.csv`. We could add an example file called `sample_site_locations.csv` that just has the headers.

Reply:
I added these to the readme in a new section near the top.